FILE packages/relay/package.json
RANGE {"path":"packages/relay/package.json","old_lines":[15,18],"new_lines":[15,19],"diff_block_lines":[5,10],"hunk_header":"@@ -15,4 +15,5 @@","substantive_changed_lines":1}
@@ -15,4 +15,5 @@
       "types": "./dist/index.d.ts",
       "node": "./dist/index.js",
+      "import": "./src/index.ts",
       "default": "./src/index.ts"
     },
